server/course/course.controller.js

```diff
@@ -54,6 +54,11 @@ function patch({ body, course, user }, res) {
 
 function remove({ course, user }, res) {
   return course.destroy({ context: { userId: user.id } })
+    .then(course => {
+      const deprecatedAt = course.dataValues.deleted_at;
```

Comment:
If anyone knows why is only snake case "deleted_at" field populated and the change only get's reflected in dataValues (on returned instance meta deleted_at doesn't get populated, nor aliased), help would be appreciated. @vladimyr

Reply:
This is resolved, by passing ```returning: true``` in options instance gets returned with exact data.